src/modules/Thread/context/hooks/useSendMultipleFilesMessage.ts

```diff
@@ -0,0 +1,7 @@
+export * from '../../../Channel/context/hooks/useSendMultipleFilesMessage';
+/**
+ * We have to reduce the duplicated logics between the modules Channel and Thread.
+ * We need to make a new directory place for both and bring the duplicated logics
+ * to there.
+ * The most of the custom hooks should be there.
+ */
```

Comment:
Just verify if Message actions are same for channel and thread?

Reply:
Actually, if you use `pubsub` instead of message actions, this concern wont exist